std/experimental/ndslice/selection.d

```diff
@@ -707,6 +708,58 @@ pure nothrow unittest
 }
 
 /++
+Returns an m-dimensional slice composed of n-dimensional identical slices,
+where `m` is a number of dimension passed to the `clones`.
+`clones` can be generalized with other selectors.
+
+Params:
+    lengths = dimensions of clones
+Returns:
+    packed `M`-dimensional slice composed of `N`-dimensional slices
```

Comment:
1. It is slice composed of slices. It should be unpacked to be N + M slices.
2. `+ 1`  is true only if you pass single length, but if you pass M lengths..